package.json

```diff
@@ -18,30 +18,37 @@
     "node": ">= 5.0.0"
   },
   "author": "Roxanne Panthaky (https://github.com/rox163)",
-  "contributors": [],
+  "contributors": [
+    "Matthew Dahl (https://github.com/sandersky)",
+    "Quincy Le (https://github.com/quincyle)",
+    "Olufemi Adeojo (https://github.com/oadeojo)",
+    "Gosia Hyndman (https://github.com/vesper2000)",
+    "Michael Carroll (https://github.com/juwara0)"
+  ],
   "license": "MIT",
   "devDependencies": {
     "broccoli-asset-rev": "^2.4.2",
     "ember-block-slots": "^1.1.0",
     "ember-browserify": "1.1.11",
-    "ember-cli": "^2.4.2",
+    "ember-cli": "~2.6.0",
     "ember-cli-app-version": "1.0.0",
-    "ember-cli-blanket": "0.9.1",
-    "ember-cli-content-security-policy": "0.5.0",
+    "ember-cli-blanket": "0.9.5",
     "ember-cli-dependency-checker": "^1.2.0",
     "ember-cli-htmlbars-inline-precompile": "0.3.1",
-    "ember-cli-ic-ajax": "0.2.4",
     "ember-cli-inject-live-reload": "1.4.0",
     "ember-cli-mirage": "0.1.13",
-    "ember-cli-mocha": "0.10.1",
+    "ember-cli-mocha": "0.10.4",
     "ember-cli-notifications": "3.2.0",
+    "ember-cli-release": "0.2.9",
+    "ember-cli-sri": "^2.1.0",
     "ember-cli-uglify": "1.2.0",
     "ember-computed-decorators": "0.2.2",
-    "ember-data": "^2.4.0",
+    "ember-data": "~2.6.0",
     "ember-disable-proxy-controllers": "1.0.1",
     "ember-export-application-global": "1.0.5",
     "ember-frost-core": ">=0.20.0 <1.0.0",
```

Comment:
Same here, dependencies - and we should adjust the range in the blueprint to match `<1.0.0`